### ISSUE TITLE
Font import/usage errors after converting a project from Godot 3 to Godot 4

### ISSUE THREAD

**Issue by toby3d** [godotengine/godot#76131]
### Godot version

v4.0.rc.arch_linux [c7aadab15]

### System information

Linux 6.2.10-1-MANJARO x86_64 GNU/Linux, GeForce RTX 3080 Mobile / Max-Q 8GB/16GB

### Issue description

After migrating a project from Godot 3 to Godot 4, I encountered that the fonts previously used in Label nodes can no longer be used. Reimporting resources with a import properties reset doesn't help. The console generates the following error log on project startup and on attempts to install a font:
```
Godot Engine v4.0.rc.arch_linux (c) 2007-present Juan Linietsky, Ariel Manzur & Godot Contributors.
  modules/gltf/register_types.cpp:70 - Blend file import is enabled in the project settings, but no Blender path is configured in the editor settings. Blend files will not be imported.
--- Debug adapter server started ---
--- GDScript language server started ---
  FreeType: Error loading font: ''.
  modules/text_server_adv/text_server_adv.cpp:1804 - Condition "!_ensure_cache_for_size(fd, size)" is true. Returning: nullptr
  FreeType: Error loading font: ''.
  modules/text_server_adv/text_server_adv.cpp:2431 - Condition "!_ensure_cache_for_size(fd, size)" is true. Returning: 0.0
  modules/text_server_adv/text_server_adv.cpp:5326 - Condition "hb_font == nullptr" is true.
  FreeType: Error loading font: ''.
  modules/text_server_adv/text_server_adv.cpp:2319 - Condition "!_ensure_cache_for_size(fd, size)" is true. Returning: 0.0
  FreeType: Error loading font: ''.
  modules/text_server_adv/text_server_adv.cpp:2345 - Condition "!_ensure_cache_for_size(fd, size)" is true. Returning: 0.0
  FreeType: Error loading font: ''.
  modules/text_server_adv/text_server_adv.cpp:2319 - Condition "!_ensure_cache_for_size(fd, size)" is true. Returning: 0.0
  FreeType: Error loading font: ''.
  modules/text_server_adv/text_server_adv.cpp:2345 - Condition "!_ensure_cache_for_size(fd, size)" is true. Returning: 0.0
Set theme_override_fonts/font
  FreeType: Error loading font: ''.
  modules/text_server_adv/text_server_adv.cpp:1804 - Condition "!_ensure_cache_for_size(fd, size)" is true. Returning: nullptr
  FreeType: Error loading font: ''.
  modules/text_server_adv/text_server_adv.cpp:2431 - Condition "!_ensure_cache_for_size(fd, size)" is true. Returning: 0.0
  modules/text_server_adv/text_server_adv.cpp:5326 - Condition "hb_font == nullptr" is true.
  FreeType: Error loading font: ''.
  modules/text_server_adv/text_server_adv.cpp:2319 - Condition "!_ensure_cache_for_size(fd, size)" is true. Returning: 0.0
  FreeType: Error loading font: ''.
  modules/text_server_adv/text_server_adv.cpp:2345 - Condition "!_ensure_cache_for_size(fd, size)" is true. Returning: 0.0
```

### Steps to reproduce

1. Open the attached project
2. Import any font from the fonts directory into the Label node

### Minimal reproduction project

[bug.zip](https://github.com/godotengine/godot/files/11242287/bug.zip)

**Comment by Wani4ka** [comment]
The font files from the project are invalid themselves. Can't open them even with default Windows Font Viewer:

![image](https://user-images.githubusercontent.com/31739611/232739683-329c61e4-9c50-49d4-8c65-59ffb82bb244.png)

Maybe try adding original font files once again? Looks like they were corrupted somehow during migration

**Comment by toby3d** [comment]
With Git, I rolled back to the project state before migration and tried to open it with Godot 3.5.2 - fonts work and are displayed in Label nodes as they should be.

I tried to open this state in Godot 4 and agreed with project conversion. The editor hangs while opening the main scene, but manages to update the cache of font imports. By copying them into a new empty project I checked that they open.

Probably the files were corrupted in one of the last beta versions of Godot 4 I tried to migrate a project in, due to some bug that has already been fixed in the current version.

I will close the issue, since manually updating the font files solves the problem.
